repose-aggregator/artifacts/valve/src/main/scala/org/openrepose/valve/ReposeJettyServer.scala

```diff
@@ -126,37 +126,51 @@ class ReposeJettyServer(val nodeContext: AbstractApplicationContext,
 
         //Handle the Protocols and Ciphers
         //varargs are annoying, so lets deal with this using the scala methods
+        // From https://www.eclipse.org/jetty/documentation/9.4.x/configuring-ssl.html
+        // "When working with Includes / Excludes, it is important to know that Excludes will always win."
+        // TODO: The SslContextFactory's set Include/Exclude Protocols/CipherSuites override the built-in defaults.
+        // TODO: We may want to change at least the Excludes back to add's for v9.x.
         import scala.collection.JavaConversions._
-        Option(ssl.getExcludedProtocols).foreach(xp => cf.addExcludeProtocols(xp.getProtocol.toList: _*))
-        Option(ssl.getIncludedProtocols).foreach { ip =>
-          if (ip.getProtocol.nonEmpty) {
-            cf.setIncludeProtocols(ip.getProtocol.toList: _*)
-          }
-
-        }
-        Option(ssl.getExcludedCiphers).foreach { xc =>
-          cf.addExcludeCipherSuites(xc.getCipher.toList: _*)
-        }
-
-        Option(ssl.getIncludedCiphers).foreach { ic =>
-          if (ic.getCipher.nonEmpty) {
-            cf.setIncludeCipherSuites(ic.getCipher.toList: _*)
+        Option(ssl.getIncludedProtocols) foreach { ip => cf.setIncludeProtocols(ip.getProtocol.toList: _*) }
+        Option(ssl.getExcludedProtocols) foreach { ep => cf.addExcludeProtocols(ep.getProtocol.toList: _*) }
```

Comment:
I think we can set this rather than add, right? In 9.2.0, there are no default includes or excludes as far as I can tell, so it should be safe to do the following:
```
cf.setIncludeProtocols()
cf.addExcludeProtocols()
cf.setIncludeCipherSuites()
cf.setExcludeCipherSuites()
Option(ssl.getIncludedProtocols) foreach { ip => cf.setIncludeProtocols(ip.getProtocol.toList: _*) }
Option(ssl.getIncludedProtocols) foreach { ip => cf.setExcludeProtocols(ep.getProtocol.toList: _*) }
Option(ssl.getIncludedCiphers) foreach { ic => cf.setIncludeCipherSuites(ic.getCipher.toList: _*) }
Option(ssl. getExcludedCiphers) foreach { ic => cf.setExcludeCipherSuites(ec.getCipher.toList: _*) }
```

That gives us 9.2.0 behavior in 9.3.0 and onward.

In the future, we may want to consider using the Jetty defaults by default or even making our own defaults if we want to decouple with Jetty.

Reply:
Added.